src/detect-byte-extract.c

```diff
@@ -629,7 +629,7 @@ static int DetectByteExtractSetup(DetectEngineCtx *de_ctx, Signature *s, const c
  error:
     DetectByteExtractFree(de_ctx, data);
     return ret;
-}
```

Comment:
unrelated change, also it's wrong

Reply:
when I send the PR, the test complains code style error, so I followed the clang-format to format the code, it seems these changes come from clang-format. I will see if I can remove them